src/elements/fields/TextField.jsx

```diff
@@ -89,6 +90,15 @@ function TextField({
 }) {
   const servar = element.servar;
   const inputType = fieldProps.as === 'textarea' ? 'textarea' : 'input';
+
+  useHotkeys(
+    'enter',
+    (e) => (inputType === 'textarea' && e.stopPropagation()),
```

Comment:
why not just enable this only on inputs and remove this if check? also this is currently buggy, enter will submit text areas even though it shouldn't. did you test this code path?

Reply:
Modified this logic to capture only the text area. `enter` events and stop the propagation.